content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -313,7 +356,7 @@ admission policy are handled. Allowed values are `Ignore` or `Fail`.
 Note that the `failurePolicy` is defined inside `ValidatingAdmissionPolicy`:
 
 ```yaml
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?